cognite/client/data_classes/_base.py

```diff
@@ -443,8 +443,8 @@ def __getattribute__(self, item: Any) -> Any:
     @classmethod
     def _load(cls: Type[T_CogniteFilter], resource: Union[Dict, str]) -> T_CogniteFilter:
         if isinstance(resource, str):
-            return cls._load(json.loads(resource))
-        elif isinstance(resource, Dict):
+            resource = json.loads(resource)
```

Comment:
This doesn't get returned now?

Reply:
It does, since I also changed the `elif` below to `if`. But - I thought I had reversed this change. It dawned on me that a subclass might do something with the dict, but not a string, and so having this be recursive is probably correct. Will revert 👍